test/bigtest/mocks/form-config-samples.js

```diff
@@ -1049,7 +1049,7 @@ export const formConfigSamples = [{
                 fields: 'yearCaption',
                 fieldsPath: 'mappingDetails.mappingFields[18].subfields',
                 incrementalField: 'order',
-                legend: 'ui-data-import.settings.mappingProfiles.map.item.enumerationData.field.yearCaption.legend',
```

Comment:
If this translation is not used anywhere else remove it from `en.json` file as well, please